### DISCUSSION TITLE
customize css and bootstrap use

### DISCUSSION THREAD

**Discussion by benoit-hash** [kaedroho/django-bridge#52] [Q&A] [answered]
Hello,
First, I would like to thank you for this beautiful and very helpful project.
I am building my new application project on django-bridge but I don't know if there is an easy way to customize css or the base template for my application in order to be closer to my needs.
From my understanding, every page is based on django_bridge/bootstrap.html template. Is there a way to overwrite this ?
Thanks

**Comment by kaedroho** [comment, marked as answer]
My preferred way to do this would be to use a styling framework on the frontend like [styled-components](https://styled-components.com/) or [CSS modules](https://medium.com/@ralph1786/using-css-modules-in-react-app-c2079eadbb87). These work well with `django-bridge` along with other styling frameworks that support React.

Adding a CSS file manually to the boostrap template is also possible: You can override `django_bridge/bootstrap.html` by creating a template with the same name in a templates folder in your project and adding the following content:

```html+django
{% extends "django_bridge/bootstrap.html" %}

{% block extra_head %}
  <link rel="stylesheet" href="LINK TO CUSTOM CSS HERE"/>
{% endblock %}
```

You can find out more about overriding Django templates here: https://docs.djangoproject.com/en/5.1/howto/overriding-templates/
